readthedocs/telemetry/collectors.py

```diff
@@ -121,9 +128,44 @@ def _get_all_conda_packages(self):
             return packages
         return []
 
-    def _get_pip_packages(self, include_all=False):
+    def _get_user_pip_packages(self):
+        """
+        Get all the packages to be installed defined by the user.
         """
-        Get all the packages installed by the user using pip.
+        results = []
+        for install in self.config.python.install:
+            if isinstance(install, PythonInstallRequirements):
+                if install.requirements:
+                    cmd = ["cat", install.requirements]
+                    code, stdout, _ = self.run(*cmd, cwd=self.checkout_path)
+                    df = dparse.parse(
```

Comment:
I've never heard of dparse. Probably would be useful to add a comment explaining what this is doing, or make the docstring more complete?

Reply:
I updated the docstring explaining a little more what's the content that's saved and where it comes from.